src/huggingface_hub/keras_mixin.py

```diff
@@ -533,51 +533,42 @@ def push_to_hub_keras(
 
 class KerasModelHubMixin(ModelHubMixin):
     """
-    Mixin to provide model Hub upload/download capabilities to Keras models.
-    Override this class to obtain the following internal methods:
-    - `_from_pretrained`, to load a model from the Hub or from local files.
-    - `_save_pretrained`, to save a model in the `SavedModel` format.
+    Implementation of [`ModelHubMixin`] to provide model Hub upload/download
```

Comment:
I realised that the Keras mixin had the same issue as the PyTorch one, so went ahead and fixed it too